### PR TITLE
user name (email?) --> principalId for adding/removing admin methods on bridge client

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: integration-test/src/test/java/org/sagebionetworks/IT600BridgeCommunities.java [hunk 1]
```diff
@@ -82,6 +87,13 @@ public static void beforeClass() throws Exception {
 
 		bridgeTwo = createBridgeClient(StackConfiguration.getIntegrationTestUserTwoName(),
 				StackConfiguration.getIntegrationTestUserTwoPassword());
+		
+		// Calling createSynapse(bridgeTwo).getUserSessionData().getProfile().getOwnerId();
+		// generates an "invalid session token" error. Do not know why.
+		SynapseClientImpl synapse = createSynapse();
```
Comment (marcel-blonk) [inline integration-test/src/test/java/org/sagebionetworks/IT600BridgeCommunities.java]: That invalid session token needs to be figured out. The test here is partially to test the creation of a synapse client based on a bridge client. We can't just taking that test out. It'll come back to bite us.

Comment (alxdarksage) [inline integration-test/src/test/java/org/sagebionetworks/IT600BridgeCommunities.java]: Going to ask Joseph about this tomorrow: I think it's a bug.

Comment (kaysoky) [inline integration-test/src/test/java/org/sagebionetworks/IT600BridgeCommunities.java]: Let me replicate it and see what I find.

Comment (kaysoky) [inline integration-test/src/test/java/org/sagebionetworks/IT600BridgeCommunities.java]: Not a bug in the auth services.  You're missing a line in the createSynapse(BridgeClient bridge) method

``` java
synapse.setAuthEndpoint(StackConfiguration.getAuthenticationServicePrivateEndpoint());
```

**Pull request merged**